Audio/AudioEngine.cpp

```diff
@@ -1467,38 +1475,117 @@ X3DAUDIO_HANDLE& AudioEngine::Get3DHandle() const noexcept
 #endif
 #include <Windows.Devices.Enumeration.h>
 #pragma warning(pop)
+
 #include <wrl.h>
-#endif
+
+namespace
+{
+    uint16_t GetDeviceBitDepth(const wchar_t*)
+    {
+        return 16; // This is a guess as there's no way to get this data via the Windows Runtime APIs
```

Comment:
I'm trying to see if there's a way I can get the correct bit-depth out of the Windows Runtime APIs. It's looking like it's possible, but of course a huge PITA.

Reply:
This has been implemented for Windows Runtime enumeration as well.